File: Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/repositories/models/response/QuestionsResponse.java
package com.dreams.interviewprepapp.repositories.models.response;

import com.dreams.interviewprepapp.repositories.models.Question;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Tc2r on 5/19/2017.
 * <p>
 * Description:
 */

public class QuestionsResponse {
    @SerializedName("questions")
    private ArrayList<Question> questionsList;

    public ArrayList<Question> getQuestionsList() {
        return questionsList;
    }
}

File: Interview_Prep_App/native_kotlin/view_binding/src/main/java/com/interviewprep/kotlinviewbinding/repository/models/response/QuestionsResponse.kt
package com.interviewprep.kotlinviewbinding.repository.models.response

import com.google.gson.annotations.SerializedName
import com.interviewprep.kotlinviewbinding.repository.models.Question

data class QuestionsResponse(
    @SerializedName("questions")
    var questions: ArrayList<Question>
)
